### ISSUE TITLE
Training agent using gpu

### ISSUE THREAD

**Issue by mancho1987** [hsahovic/poke-env#102]
Hi,

Is it possible to train the agent using gpu?

**Comment by hsahovic** [comment]
Hi @mancho1987,

You can use GPUs to train models used by agents, but the specifics will depend on your specific choices of algorithm and computing framework.
That being said, you will still have a sizable cpu overhead consisting of node + the python client.

**Comment by mancho1987** [comment]
Thanks for the answer! Could you please give me an example as how it would be done in the dqn agent example you provide?

**Comment by hsahovic** [comment]
My understanding is that `keras-rl2` uses `tensorflow.keras` under the hood, so if your installation of `tensorflow` takes advantage of your gpu (eg. you installed `tensorflow-gpu` and all the relevant nvidia libraries), the dqn agent example should use your gpu for training and inference without requiring any code modification.
